config/locales/en.yml

```diff
@@ -175,6 +175,7 @@ en:
     show_current_version_with_authorship_highlighting: 'Current Version w/ Authorship Highlighting'
     title: Title
     tracked: Tracked
+    tracked_doc: The status of whether or not an article has been tracked. The course stats depend only on tracked articles so articles deemed not relevant to the course can be untracked here and the stats will reflect that the next time the course is updated.
```

Comment:
`Only tracked articles contribute to the the statistics for this program. When articles are marked as untracked, they will be removed from the statistics during the next data update.`